src/AzureStack/ComputeAdmin/Compute.Admin/generate.cmd

```diff
@@ -0,0 +1,10 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+rd Generated /S /Q
+call %~dp0..\..\..\..\tools\generate.cmd azsadmin/resource-manager/compute latest deathly809 azs.compute.admin azure-rest-api-specs %CD%
```

Comment:
Please generate the code from `Azure` `master`

Reply:
Done.